helm-charts/charts/kube-starrocks/charts/starrocks/templates/fe-proxy/feproxy.yaml

```diff
@@ -0,0 +1,146 @@
+{{- if .Values.feProxy.enabled }}
+
+# create fe proxy deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "kube-starrocks.name" . }}-fe-proxy
+  namespace: {{ template "starrockscluster.namespace" . }}
+  labels:
+    {{- include "feproxy.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.feProxy.replicas }}
+  selector:
+    matchLabels:
+      {{- include "feproxy.labels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "feproxy.labels" . | nindent 8 }}
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 101
+        runAsGroup: 101
+      {{- if .Values.feProxy.nodeSelector }}
+      nodeSelector:
+        {{- toYaml .Values.feProxy.nodeSelector | nindent 8 }}
```

Comment:
probably need provide `affinity` and `tolerations` related configuration to end user as well.